src/modules/Autologin.rb

```diff
@@ -180,8 +176,7 @@ def AskForDisabling(new)
     #
     # @return Boolean
     def supported?
-      pkg_lazy_init
-      supported = DISPLAY_MANAGERS.any? { |dm| Pkg.IsSelected(dm) || Pkg.IsProvided(dm) }
+      supported = DISPLAY_MANAGERS.any? { |dm| Package.Available(dm) }
```

Comment:
@lslezak can it happen that something is selected in Pkg but not provided?

Reply:
So looks like Package Available is not perfect match. I also see that there is Package.Installed but it directly calls rpm --provides. So neither of it is good match. Is there a method that ensure that target is initialized and provide similar functionality? Ideally some higher level API then this low level Pkg calls?

Reply:
Hmm, the Package APIs are here https://apitalism.herokuapp.com/?q=Package.*Available
but I don't have a simple link for the `Pkg` calls

Reply:
it lives at https://yast-pkg-bindings.surge.sh/ sadly it is not perfect or easily usable.